great_expectations/datasource/fluent/interfaces.py

```diff
@@ -56,19 +56,26 @@
     AbstractSetIntStr = AbstractSet[Union[int, str]]
     # TODO: We should try to import the annotations from core.batch so we no longer need to call
     #  Batch.update_forward_refs() before instantiation.
-    from great_expectations.core.batch import (
+    from great_expectations.core.batch import (  # noqa: TCH004
         BatchData,
         BatchDefinition,
         BatchMarkers,
     )
     from great_expectations.core.config_provider import _ConfigurationProvider
-    from great_expectations.data_context import AbstractDataContext as GXDataContext
+    from great_expectations.data_context import (  # noqa: TCH004
+        AbstractDataContext as GXDataContext,
+    )
     from great_expectations.datasource.data_connector.batch_filter import BatchSlice
-    from great_expectations.datasource.fluent import BatchRequest, BatchRequestOptions
-    from great_expectations.datasource.fluent.data_asset.data_connector import (
+    from great_expectations.datasource.fluent import (  # noqa: TCH004
+        BatchRequest,
+        BatchRequestOptions,
+    )
+    from great_expectations.datasource.fluent.data_asset.data_connector import (  # noqa: TCH004
         DataConnector,
     )
-    from great_expectations.datasource.fluent.type_lookup import TypeLookup
+    from great_expectations.datasource.fluent.type_lookup import (  # noqa: TCH004
+        TypeLookup,
+    )
```

Comment:
@Shinnnyshinshin these noqa comments should not have been needed.
I suspect your local version of ruff is out of date.